<element name="camera" required="0">
  <element name="horizontal_fov" required="1">
   <attribute name="angle" type="double" default="1.047" required="1"/>
  </element> <!-- End Horizontal_FOV -->
  <element name="image" required="1">
    <attribute name="width" type="int" default="320" required="1"/>
    <attribute name="height" type="int" default="240" required="1"/>
    <attribute name="format" type="string" default="R8G8B8" required="0"/>
  </element> <!-- End Image -->
  <element name="clip" required="1">
    <attribute name="near" type="double" default=".1" required="1"/>
    <attribute name="far" type="double" default="100" required="1"/>
  </element> <!-- End Clip -->
  <element name="save" required="0">
    <attribute name="enabled" type="bool" default="false" required="1"/>
    <attribute name="path" type="string" default="__default__" required="1"/>
  </element> <!-- End Save -->
  <element name="depth_camera" required="0">
    <attribute name="output" type="string" default="depths" required="1"/>
  </element> <!-- End Save -->
</element> <!-- End Camera -->
